docs_src/src/pages/documentation/api_reference/getting_started.mdx

```diff
@@ -548,6 +548,7 @@ Either, by using the `headers` field in the `Request` object:
       print("These are the request headers: ", headers)
 
       headers.set("modified", "modified_value")
+      # We can also write headers["modified"] = "modified_value"
```

Comment:
Hey @AntoineRR 👋 

You can set a different modified value here instead of the comment😄

Reply:
True, I did this